foodie-application/src/main/java/com/foodie/FoodieApplication.java

```diff
@@ -2,12 +2,17 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+
 
 @SpringBootApplication
+@ComponentScan
```

Comment:
what is the need for having this annotation here

Reply:
had a lot of conflicts to resolve, thanks for highlighting it. 
will confirm once I check all the files. then we can resolve this thread.